i18n/web/index.ts

```diff
@@ -106,11 +106,40 @@ export type WebTranslations = {
     header: string
     tableHeader1: string
     tableHeader2: string
-    tableHeader3: string
-    tableHeader4: string
     tableTotalAmount: string
+    whatYouToldUs: string,
+    youMayBeEligible: string
+    basedOnYourInfo: string
+    yourEstimatedTotal: string
+    basedOnYourInfoTotal: string
+    nextSteps: string
+    youMayNotBeEligible: string
+  }
+  resultsQuestions: {
```

Comment:
for translations we already have elsewhere, such as the question text and the legal status options, please refer to the original text rather than duplicating it here. you can see how we've done this elsewhere here: https://github.com/DTS-STN/eligibility-estimator/blob/develop/i18n/web/en.ts#L116

at some point soon I will refactor and clean up the translations so there's just one and we don't need to worry about this separation anymore, but for now please avoid duplication

Reply:
Agree on the duplication but the figma mockup have way shorter version of the text than the questions page